packages/web-components/fast-element/src/attributes.ts

```diff
@@ -1,6 +1,6 @@
-import { Accessor, Observable } from "./observation/observable";
-import { DOM } from "./dom";
-import { Notifier } from "./observation/notifier";
+import { Accessor, Observable } from "./observation/observable.js";
```

Comment:
Is there any configuration option we can use in order to sidestep adding the filename extension?

Reply:
Not that I know of. There have been a couple of long discussion threads in TS with no clear action yet though.